src/ott/datasets.py

```diff
@@ -152,3 +319,52 @@ def create_gaussian_mixture_samplers(
   )
   dim_data = 2
   return train_dataset, valid_dataset, dim_data
+
+
+def create_sklearn_samplers(
+    source_kwargs: Mapping[str, Any] = MappingProxyType({}),
+    target_kwargs: Mapping[str, Any] = MappingProxyType({}),
+    train_batch_size: int = 256,
+    valid_batch_size: int = 256,
+    rng: Optional[jax.Array] = None,
+) -> Tuple[Dataset, Dataset, int]:
+  """Sklearn samplers for :class:`~ott.solvers.nn.neuraldual.W2NeuralDual`.
+
+  Args:
+  source_kwargs: kwargs to initialize source sampler
```

Comment:
Please use:

Keyword arguments for the source :class:`~ott.datasets.SklearnDistribution`.

(Similarly for the target.)